dashboard/app/models/concerns/serialized_properties.rb

```diff
@@ -14,6 +14,11 @@ def assign_attributes(new_attributes)
     init_properties
     attributes = new_attributes.stringify_keys
     new_properties = attributes.delete('properties').try(:stringify_keys!)
+
+    concept_difficulty_attributes = attributes.delete('level_concept_difficulty')
+    assign_nested_attributes_for_one_to_one_association(:level_concept_difficulty,
+        concept_difficulty_attributes) if concept_difficulty_attributes
```

Comment:
I'd really like to try and find another place to include this attribute-specific logic. SerializedProperties was written as a generic Concern (roughly similar to the [ActiveRecord::Store](http://api.rubyonrails.org/classes/ActiveRecord/Store.html) implementation, which I would have used instead had I known it existed at the time), but making it recognize `level_concept_difficulty` specifically breaks that abstraction.